idb/postgres/postgres.go

```diff
@@ -2117,9 +2120,12 @@ func (db *IndexerDb) yieldAccountsThread(req *getAccountsRequest) {
 						Total:         ap.Total,
 						Decimals:      uint64(ap.Decimals),
 						DefaultFrozen: boolPtr(ap.DefaultFrozen),
-						UnitName:      stringPtr(ap.UnitName),
-						Name:          stringPtr(ap.AssetName),
-						Url:           stringPtr(ap.URL),
+						UnitName:      stringPtr(ap.AssetParams.UnitName),
```

Comment:
It's not incorrect, but why do you use `ap.AssetParams.UnitName` instead of `ap.UnitName` ? i.e. it's an embedded struct

Reply:
Thanks, I missed this one. Previously I had `UnitName` defined twice, then I renamed the second one to `UnitNameBytes`.